examples/oc-phoenix/storage-oc.toml

```diff
@@ -22,7 +22,8 @@ data_server_url = "http://localhost:11001/data"
 
 [grpc.services.storageprovider.drivers.owncloud]
 datadirectory = "/var/tmp/reva/data"
-
+redis = "redis:6379"
+userprovidersvc = "localhost:18000"
```

Comment:
Setting this will save a trip to the gateway. When left out it should fall back to the `gatewaysvc` in the `[shared]` section and work just as well.